### ISSUE TITLE
PDF.js Message: PDFDocument: stream must have data

### ISSUE THREAD

**Issue by carnage101** [izuzak/atom-pdf-view#209]
Working on a latex project using [`atom-latex`](https://atom.io/packages/latex) and `texlive2018`

- `main.tex`

````
\documentclass[options]{extbook}
  \include{./preamble.tex}
  
\begin{document}
  \frontmatter
  \toc
\mainmatter
  \include{./chapter-1}
  \include{./chapter-2}
  \include{./...}
  \include{./chapter-14}
\end{document}
````

- `chapter#.tex`

````
\chapter{}
  \section{}
    \subsubsection{}
      \input{./--/tables}
      \input{./--/figures}
````

After building the pdf for the whole book I was trying to use `\includeonly{chapter-1.tex}` in line 3 of `main.tex` to recompile only one `.tex` subfile in the pdf (instead of the whole book) in order to save on compiling time. When I build the project `atom-latex` reports it built successfully while `pdf-view` pane reports this error:

````
An error occured while loading the PDF.

PDF.js v1.9.426 (build: 2558a58d)
Message: PDFDocument: stream must have data
````

Possibly a `pdf-view` issue?

PS: If it helps, I'm using `\newcommand*\ExpandableInput[1]{\@@input#1 }` in `preamble.tex` in order to `\input` tables and figures from within an `\include{chapter-1}`.

**Comment by izuzak** [comment]
Thanks for opening an issue, @carnage101. Can you share the generated PDF so that I can try reproducing the problem?

**Comment by carnage101** [comment]
Hi @izuzak . I can't share the PDF. I could set up dummy files in a documents structure similar to the one I'm using so that you can reproduce the issue. Would that work for you?

**Comment by izuzak** [comment]
> Hi @izuzak . I can't share the PDF. I could set up dummy files in a documents structure similar to the one I'm using and share that, so that you can reproduce the issue. Would that work for you?

No, that wouldn't work for me. This package works with PDFs, not with latex files. So, you'd need provide a PDF that fails to load for you to demonstrate a problem (which I'd try to reproduce). 

Why can't you share the PDF? It doesn't have to have any real content in it, feel free to replace all the content with lorem ipsum text if you'd like

**Comment by carnage101** [comment]
Alright I meant the same thing. I'll provide a dummy pdf that fails to load under the same conditions in a couple of minutes.

**Comment by carnage101** [comment]
[issue209_main.zip](https://github.com/izuzak/atom-pdf-view/files/1968263/issue209_main.zip)

**Comment by izuzak** [comment]
Thanks. As far as I can tell, the PDF in that ZIP has 0 bytes and can't even be opened by MacOS preview or Chrome's PDF viewer. Are you able to open that PDF with some other PDF viewer?

**Comment by carnage101** [comment]
Yeah, no sorry if I wasted your time. The issue was in the syntax of `\includeonly{chapterfilename}` which would cause a code error and a blank pdf. Apparently [TeX wiki](https://en.wikibooks.org/wiki/TeX/includeonly) reports the syntax for that command wrongly.....
